src/main/java/org/candlepin/subscriptions/subscription/SubscriptionServiceConfiguration.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2021 Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.subscriptions.subscription;
+
+import org.candlepin.subscriptions.http.HttpClientProperties;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * The configuration class for Subscription Service related client support.
+ */
+@Configuration
+@ComponentScan("org.candlepin.subscriptions.subscription")
+public class SubscriptionServiceConfiguration {
+
+    @Bean
+    @Qualifier("subscription")
+    @ConfigurationProperties(prefix = "rhsm-subscriptions.subscription")
```

Comment:
Can you please add some properties to `application-worker.yaml` for this with environment vars and sane defaults? (Could reference `application-rhsm-conduit.yaml` -> `rhsm-conduit.rhsm`)